functionary/ui/templates/partials/tasks/list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "partials/table.html" %}
+{% block table_headers %}
+    <th>Created at</th>
```

Comment:
I think the order of these columns should go:

* Function
* Status
* Creator
* Created At

With the function name just linking over the task.  I don't know that there needs to be a link over to the function detail page from here.  We can probably find a convenient home for that link on the Task detail page eventually.

Same for the Builds table (Package > Status > Creator > Created At).